app/views/filter/index.html.erb

```diff
@@ -24,4 +25,20 @@
   </h2>
 </div>
 
+<% if params[:overwatch_2] %>
+  <div class="well well--dark b-overwatch-2 mb-1/2">
+    <div>
+      <h3 class="mt-0 mb-0">
+        <strong><%= t("banner.overwatch_2.title_html") %></strong>
+      </h3>
+
+      <div class="mt-1/4 text-small">
+        These Workshop codes were tagged as being importable within Overwatch 2.
+        It is possible that codes may have been invalidated, since Overwatch 2 has not been fully released yet.
+        An import code that was created in Overwatch 2 will not work in Overwatch 1.
```

Comment:
```suggestion
        These Workshop codes were tagged as being importable within the Overwatch 2 Beta.
        It is possible that codes may have been invalidated, since the Overwatch 2 Beta is under active development.
        An import code that was created in Overwatch 2 Beta will not work in Overwatch 1.
```